code/obj/flock/structure/rift.dm

```diff
@@ -56,3 +56,8 @@
 	else
 		var/severity = round(((build_time - elapsed)/build_time) * 5)
 		animate_shake(src, severity, severity)
+
+/obj/flock_structure/rift/disposing()
+	if (!src.flock?.flockmind?.started)
+		src.flock?.flockmind?.death()
+	..()
```

Comment:
```suggestion
	..()
	if (!src.flock?.flockmind?.started)
		src.flock?.flockmind?.death()
```
Might actually think parent should be called first if nothing wrong with it, since death() will gib all builidings disposing will be called twice, but minor thing, and calling parent first should hopefully clean it up before death

Reply:
Just tested it, and this causes the proc to stop after the parent call and not do anything (since the rift just got disposed)

Reply:
Yeah I was thinking about it a bit more, think might have to get Flockmind in a variable, call parent, and call death on the Flockmind